src/config/default-config.js

```diff
@@ -113,9 +113,9 @@ if (config.clusterEnv === 'development') {
     s3ForcePathStyle: true,
   });
 
-  // remove this line when the new gem2s-endpoint is merged and thus the production/pipeline.yaml is
-  // updated to new name pipeline-runner
-  config.pipelineInstanceConfigUrl = 'https://raw.githubusercontent.com/biomage-ltd/iac/master/releases/production/pipeline.yaml';
+  // TODO: remove this line when scaling work is merged and thus the production/pipeline.yaml is
```

Comment:
This should be done before closing this PR or later on? If it should be done later we should add a ticket so we don';t forget.

If it is before closing this PR we can use this comment as reminder

Reply:
just before merging the PR